resources/tentacle.rb

```diff
@@ -29,6 +29,11 @@
 attribute :app_path, kind_of: String, default: 'C:\Octopus\Applications'
 attribute :trusted_cert, kind_of: String
 attribute :port, kind_of: Fixnum, default: 10_933
+attribute :polling_port, kind_of: Fixnum, default: 10_943
```

Comment:
ummm not sure.  I also think there are issues with lazy evaluation on earlier versions of chef-client.  Maybe the best thing to do is to set the value to nil and if its nil just default it in the resource?